include/api.php

```diff
@@ -60,16 +70,21 @@ function pll_current_language( $field = 'slug' ) {
  * @return string|PLL_Language|false The requested field for the default language.
  */
 function pll_default_language( $field = 'slug' ) {
-	if ( isset( PLL()->options['default_lang'] ) ) {
-		$lang = PLL()->model->get_language( PLL()->options['default_lang'] );
-		if ( $lang ) {
-			if ( OBJECT === $field ) {
-				return $lang;
-			}
-			return isset( $lang->$field ) ? $lang->$field : false;
-		}
+	if ( empty( PLL()->options['default_lang'] ) ) {
+		return false;
+	}
+
+	$lang = PLL()->model->get_language( PLL()->options['default_lang'] );
+
+	if ( empty( $lang ) ) {
+		return false;
+	}
+
+	if ( OBJECT === $field ) {
+		return $lang;
 	}
-	return false;
+
+	return isset( $lang->$field ) ? $lang->$field : false;
```

Comment:
I prefer `isset()` because it's false when the property exists but is not set.
That said, you are right, the function does not make the difference between a wrong property and a non-existing default language.